app/presenters/post_presenter.rb

```diff
@@ -16,6 +16,26 @@ def as_json(_options={})
          .merge(non_directly_retrieved_attributes)
   end
 
+  def as_api_response
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for as_api_response is too high. [22/20]